app/usecases/sendAbsent.test.js

```diff
@@ -0,0 +1,74 @@
+
+const messageResponse = require("../../__fixtures__/watsonAssistant/watsonContextResponse");
+const messageResponseAllTimes = require("../../__fixtures__/watsonAssistant/AllTimeResponse");
+const messageResponseNoTimes = require("../../__fixtures__/watsonAssistant/NoTimeResponse");
+let sendAbsent;
+
+beforeEach(() => {
+  sendAbsent= require("./sendAbsent")();
+  jest.resetModules();
+});
```

Comment:
Sollte das hier nicht in den describe-Block? Siehe convo in WhatsApp mit Stefan.

Reply:
+1